bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/handler/EnOceanBridgeHandler.java

```diff
@@ -165,17 +165,14 @@ public void responseReceived(BaseResponse response) {
     public void initialize() {
         updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_PENDING, "trying to connect to gateway...");
 
-        ScheduledFuture<?> localConnectorTask = connectorTask;
-        if (localConnectorTask == null || localConnectorTask.isDone()) {
-            localConnectorTask = scheduler.scheduleWithFixedDelay(new Runnable() {
-                @Override
-                public void run() {
-                    if (thing.getStatus() != ThingStatus.ONLINE) {
-                        initTransceiver();
-                    }
+        connectorTask = scheduler.scheduleWithFixedDelay(new Runnable() {
```

Comment:
I think the only change needed is adding a line:

```suggestion
        connectorTask = localConnectorTask
```

Reply:
No, `initialize()` shouldn't be checking any existing jobs as there shouldn't be any. I guess this was somehow trying to compensate for wild-running jobs after `dispose()`?

Reply:
While it might be more robust to check, it would also hide an underlying problem. So yeah totally agree.